dev/release/02-source-test.rb

```diff
@@ -0,0 +1,88 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require "rexml/document"
+
+class SourceTest < Test::Unit::TestCase
+  include GitRunnable
+
+  def setup
+    @current_commit = git_current_commit
+    top_dir = Pathname(__dir__).parent.parent
+    @original_git_repository = top_dir + ".git"
+    cpp_cmake_lists = top_dir + "cpp" + "CMakeLists.txt"
+    @current_version = cpp_cmake_lists.read[/ARROW_VERSION "(.+?)"/, 1]
+    @release_version = @current_version.gsub(/-SNAPSHOT\z/, "")
+    @tag_name = "apache-arrow-#{@current_version}"
+
+    Dir.mktmpdir do |dir|
+      @test_git_repository = Pathname(dir) + "arrow"
+      git("clone", @original_git_repository.to_s, @test_git_repository.to_s)
+      Dir.chdir(@test_git_repository) do
+        yield
+      end
+    end
+  end
+
+  def prepare
+    env = {"PREPARE_DEFAULT"=>"0"}
```

Comment:
Could you put a space around "=>"? `{"PREPARE_DEFAULT" => "0"}`

Reply:
I've fixed it.